### ISSUE TITLE
package 'dyplr' is not available (for R version 3.4.3) 

### ISSUE THREAD

**Issue by evanbiederstedt** [tidyverse/dplyr#3333]
`dyplr` doesn't appear to be ready for the latest version of R. That's a bummer. 
---

Brief description of the problem

```r
 install.packages('dyplr')
--- Please select a CRAN mirror for use in this session ---
1
Warning message:
package 'dyplr' is not available (for R version 3.4.3) 
> 1
[1] 1
```

**Comment by cderv** [comment]
Have you tried with the correct spelling ? ` install.packages("dplyr")`

you ask for `dyplr` whereas it is `dplyr`. _dyplr_ does not exist. it is why you have a not available warning.

**Comment by evanbiederstedt** [comment]
It's been a long debugging session. Apologies.

Please delete. The shame would be too much to live with.

**Comment by jsan4christ** [comment]
@evanbiederstedt happens to the best of us, no worries.

**Comment by reshmamena** [comment]
I got a warning in installing igraph package in R

Warning: unable to access index for repository http://cran.rstudio.com/src/contrib:
  cannot open URL 'http://cran.rstudio.com/src/contrib/PACKAGES'
Warning: unable to access index for repository http://cran.rstudio.com/bin/windows/contrib/3.5:
  cannot open URL 'http://cran.rstudio.com/bin/windows/contrib/3.5/PACKAGES'
Warning message:
package ‘igraph’ is not available (for R version 3.5.1)

**Comment by batpigandme** [comment]
@reshmamena, this is unrelated to dplyr.
